cv_checker_backend/cv_checker_backend/middlewares/auth_middleware.py

```
@@ -0,0 +1,35 @@
+            # return {
+            #     "status": STATUS["NOT_AUTHORIZED"],
+            #     "message": "You are not authorized user. Please login to continue."
+            # }
+        response = validate_user(token, session)
+        if response["status"] is not STATUS["SUCCESS"]:
+            raise HTTPException(status_code=401, detail="Access denied. Please login to continue.")
+            # return response
+        request.state.user = response["user"]
+    except Exception as e:
+        print("Error while authenticating user from token: ", e)
+        raise HTTPException(status_code=401, detail="Access denied. Please login to continue.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)